CTW-919: Initial support for beta labels (#666)
zeus-health/ctw-component-library: .changeset/soft-carrots-sit.md

---
"@zus-health/ctw-component-library": minor
---

> Adding ability to embed "beta" labels in ZAP tabs and Patient Timeline v2 data type dropdown

Reviewer: ```suggestion
Add beta tags in the ZAP and Patient Timeline V2 dropdown.
```
